pkg/grafana/datasources_default.go

```diff
@@ -68,4 +75,51 @@ var (
 			Access: datasourceProxyAccessMode,
 		}
 	}
+
+	// Datasource for Tempo distributed tracing backend
+	DatasourceTempo = func() Datasource {
+		return Datasource{
+			Type: "tempo",
+			// We connect to the Tempo Query Frontend service to support streaming
+			URL:    "http://tempo-query-frontend.tempo.svc:3200",
+			Access: datasourceProxyAccessMode,
+			JSONData: map[string]any{
+				// Service Map configuration - generates visual service dependency maps
+				// from trace data using metrics from the connected Prometheus datasource
+				// Links to our specific Mimir instance for service map generation
+				"serviceMap": map[string]any{
+					"datasourceUid": MimirDatasourceUID,
+				},
+
+				// Node Graph configuration - enables node graph visualization
+				// for trace service dependencies
+				"nodeGraph": map[string]any{
+					"enabled": true,
+				},
```

Comment:
I can't find these options in grafana's UI when editing the Tempo datasource, so I don't understand them.

Reply:
It's under additional settings